src/assets/locales/fr.json

```diff
@@ -0,0 +1,567 @@
+  "media-browser.add_custom_object": "Custom URL or File",
+  "media-browser.add_custom_scene": "Custom Scene",
+  "media-browser.create-avatar": "Créer un Avatar",
+  "media-browser.create-scene": "Créer une scène avec {editorName}",
+  "media-browser.empty.default": "Aucun résultat. Essayer en entrant une nouvelle recherche au dessus.",
+  "media-browser.empty.favorites": "Vous n'avez pas de favorites. Cliquer sur ⭐ pour ajouter à vos favorites.",
+  "media-browser.favorites-header": "Favorite Rooms",
+  "media-browser.nav_title.avatars": "Avatars",
+  "media-browser.nav_title.gifs": "GIFs",
+  "media-browser.nav_title.images": "Images",
+  "media-browser.nav_title.poly": "Google Poly",
+  "media-browser.nav_title.scenes": "Scenes",
+  "media-browser.nav_title.sketchfab": "Sketchfab",
+  "media-browser.nav_title.twitch": "Twitch",
+  "media-browser.nav_title.videos": "Videos",
+  "media-browser.nav_title.youtube": "YouTube",
+  "media-browser.powered_by.gifs": "Chercher par Tenor",
+  "media-browser.powered_by.images": "Chercher par Bing",
+  "media-browser.powered_by.poly": "Chercher par Google",
+  "media-browser.powered_by.scenes": "Made with {editorName}",
+  "media-browser.powered_by.sketchfab": "Chercher par Sketchfab",
+  "media-browser.powered_by.twitch": "Chercher par Twitch",
+  "media-browser.powered_by.videos": "Chercher par Bing",
+  "media-browser.powered_by.youtube": "Chercher par Google",
+  "media-browser.privacy_policy": "Politique de vie privée",
+  "media-browser.report-issue": "Signaler une erreur",
+  "media-browser.search-placeholder.avatars": "Chercher des Avatars...",
+  "media-browser.search-placeholder.default": "Chercher...",
+  "media-browser.search-placeholder.gifs": "Chercher des GIFs...",
+  "media-browser.search-placeholder.images": "Chercher des Images...",
+  "media-browser.search-placeholder.poly": "Chercher des modèles Google Poly...",
+  "media-browser.search-placeholder.scenes": "Chercher des Scenes...",
+  "media-browser.search-placeholder.sketchfab": "Chercher des modèles Sketchfab...",
+  "media-browser.search-placeholder.twitch": "Chercher des streams Twitch...",
+  "media-browser.search-placeholder.videos": "Chercher des Videos...",
+  "media-browser.search-placeholder.youtube": "Chercher des videos Youtube...",
+  "media-tile.action.copy-avatar": "Copier vers mes avatars",
+  "media-tile.action.copy-scene": "Copier vers mes scenes",
+  "media-tile.action.edit-avatar": "Editer l'avatar",
+  "media-tile.action.edit-scene": "Editer la scene",
+  "media-tile.action.room-info": "Info de la Salle",
+  "media-tile.action.show-similar-avatars": "Montrer les avatars similaires",
+  "media-tile.joined-room": "Joint {relativeTime}",
+  "mic-permissions-modal.grant-mic-permissions": "Accorder les autorisations au Microphone",
+  "mic-permissions-modal.mic-access-needed": "L'acces au Microphone a eu besoin d'être entendu par les autres.{linebreak}Le Microphone peut être mis en sourdine dans la salle.",
```

Comment:
What are you trying to say ?

I cannot find the original il the `en.json` file